src/testUtils/TestRig.react.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+export var TestRigComponent = React.createClass({
```

Comment:
@BJK Why not go full ES6 syntax? Is Babel's transpiler not set up for ES6 in this repo?